spec/support/split_checkout_helper.rb

```diff
@@ -44,4 +44,13 @@ def fill_out_shipping_address
       select "New South Wales", from: "State"
     end
   end
+
+  def fill_out(notes)
+    fill_in 'Any comments or special instructions?', with: notes.to_s
+  end
+
+  def proceed_to_payment
+    click_button "Next - Payment method"
+    expect(page).to have_current_path("/checkout/payment")
```

Comment:
Checking the path is technical and users usually don't do this. Can we check for content like a headline? That also has the advantage that Capybara waits until the page is rendered.

Reply:
Replaced :+1: I guess this approach would be more suitable for a controller level spec?

Reply:
Yes, if a controller is redirecting to another path you can check for that in a controller spec. If it's rendering a view then you can check for content.

Reply:
Something to consider for #8684 :+1: